blockchains/ethereum/assets/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc/info.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "Governor DAO (GDAO)",
+    "type": "ERC20",
+    "symbol": "GDAO",
+    "decimals": 18,
+    "website": "https://governordao.org",
+    "description": "Governor DAO is a Wyoming-based Decentralized Autonomous Organization positioned as the “DAO of DAOs”. Governors offer a suite of products and services for projects looking to build out DAO qualities in their own communities. Our offerings include an industry-first sybil-resistance product for one-voice-one-vote governance, as well as governance bootstraps for new communities, consultations, and smart contract porting.\n",
+    "explorer": "https://etherscan.io/token//0x515d7e9d75e2b76db60f8a051cd890eba23286bc",
+    "status": "active",
+    "id": "0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc",
+    "links": [
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/governor_dao"
+        },
+        {
+            "name": "github",
+            "url": "https://github.com/Governor-DAO"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/GovernorProject"
+        },
+        {
+            "name": "telegram_news",
+            "url": "https://t.me/GovernorDAO_Official"
+        },
+        {
+            "name": "blog",
+            "url": "https://blog.governordao.org"
+        },
+        {
+            "name": "docs",
+            "url": "https://docs.governordao.org"
+        },
+        {
+            "name": "forum",
+            "url": "https://gdao.info"
+        },
+        {
+            "name": "discord",
+            "url": "https://discord.gg/MVh5NkN7gt"
```

Comment:
there is no current possible discord.com linking strategy - all links end in .gg